test/fixtures/experimental/decorators-2/no-computed-decorator-member/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "plugins": ["decorators2", "classProperties"],
+  "throws": "Stage 2 decorators may only be used with a class or a class method (2:2)"
```

Comment:
We will have to explain this in documentation or some other way. 

the problem is that with classProperties enabled and due to the fact that the computed property isn't considered a part of the decorator, we can't really tell if the person is trying to access decorator's member or trying to decorate a computed class property. 

So in this example it thinks that `[bizz]` is a class property.

If you disable class properties altogether you will get a syntax error at abc since it is expecting opening bracket right after the method name `[bizz]`